ngraph/frontend/paddlepaddle/src/op/assign_value.cpp

```diff
@@ -0,0 +1,33 @@
+//*****************************************************************************
```

Comment:
minor thing: this is an old-style license header version. Currently everywhere else another format is used. Would recommend to change it to new 3-lines version, at least for new files. We'll also change headers in existing file later

// Copyright (C) 2021 Intel Corporation
// SPDX-License-Identifier: Apache-2.0
//

Reply:
Got it